staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/testing/webhook_server.go

```diff
@@ -160,3 +168,38 @@ func webhookHandler(w http.ResponseWriter, r *http.Request) {
 		http.NotFound(w, r)
 	}
 }
+
+func ClockSteppingWebhookHandler(fakeClock *clock.FakeClock) func(http.ResponseWriter, *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		path := r.URL.Path
+		delay, err := strconv.ParseInt(path[strings.LastIndex(path, "/")+1:], 0, 64)
```

Comment:
Can we instead use a query parameter?

e.g. something like:

.../allow?delay=100ms

[Then you will be simply parting the r.URL.Query["delay"] ]

Reply:
Unfortunately I think its not that simple as webhooks do not support query parameters if I understand correctly.